scripts/coverage.sh

```diff
@@ -3,13 +3,25 @@
 set -e
 
 function cov {
-    pytest \
-        --ignore-glob=*/setup.py \
-        --cov ${1} \
-        --cov-append \
-        --cov-branch \
-        --cov-report='' \
-        ${1}
+    if [ ${TOX_ENV_NAME:0:4} == "py34" ]
```

Comment:
At this point, why not put the content of this script directly in the toxfile? Maybe with some clever `setenv` usage to avoid code duplication.

Reply:
I just implemented the first solution I found. Do you have any specific idea about how to put this directly in the tox file?

Reply:
I think let's pick this up as a followup task. this PR is good an getting stale so I think merging is ok.